common/app/views/fragments/analytics/google.scala.html

```diff
@@ -22,6 +36,55 @@
             .map(function(xs) { return xs[1] })[0];
     }
 
+    @* copied from common/util/cookies.js *@
+    function getCookieValues(name) {
+        var cookieVals = [],
+            nameEq = name + '=',
+            cookies = document.cookie.split(';');
+
+        cookies.forEach(function (cookie) {
+            while (cookie.charAt(0) === ' ') {
+                cookie = cookie.substring(1, cookie.length);
+            }
+
+            if (cookie.indexOf(nameEq) === 0) {
+                cookieVals.push(cookie.substring(nameEq.length, cookie.length));
+            }
+        });
+
+        return cookieVals;
+    }
+
+    @* copied from common/util/cookies.js *@
+    function getCookie(name) {
+        var cookieVal = getCookieValues(name);
+
+        if (cookieVal.length > 0) {
+            return cookieVal[0];
+        } else {
+            return null;
+        }
+    }
+
+    @* copied from common/modules/identity/api.js *@
+    function decodeBase64(str) {
+        return decodeURIComponent(escape(utilAtob(str.replace(/-/g, '+').replace(/_/g, '/').replace(/,/g, '='))));
+    }
+
+    @* based on common/modules/identity/api.js *@
+    function getIdentityIdFromCookie() {
+        var cookieData = getCookie('GU_U'),
+            userData = cookieData ? JSON.parse(decodeBase64(cookieData.split('.')[0])) : null;
+        if (userData) {
+            return userData[0];
+        } else {
+            return null;
+        }
+    }
+
+    var identityId = getIdentityIdFromCookie();
+    var isLoggedIn = (identityId !== null);
```

Comment:
isLoggedIn is a boolean regarding the users current logged in state and set on a hit level scope. The identityId is set on a user level scope. 
When looking at isLoggedIn it should be examined in conjunction with identityId. Just having identityId logged could give the impression that the user is permanently logged in (see user scope in GA) even though they may not be. 
Subscriptions and Membership have had a separate variables in place to isolate someone that has a login but is not currently logged in.
The approach above gives us a consistent way in which to approach this in Google Analytics across our properties.
Does that make sense?